### PR TITLE
Feature/routing

### PR DESCRIPTION
**Pull request by dominikakosiedowska**

### CODE REVIEW

File: src/components/MenuMain.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  handleLogoutUser = (event) => {
+    this.props.setInitialState()
+    this.props.logoutUser()
+  }
+
+  render() {
+    return (
+      <div>
+        <Menu inverted fixed={'top'}>
+          <Menu.Item as={Link} to='/dashboard' name='dashboard' className={this.props.activeElement === 'dashboard' ? 'active grey item' : ''} onClick={this.handleClick} />
```
Comment (sptaszek92) [inline src/components/MenuMain.js]: conditional classNames could be used by `classnames` package :)
take a look [here](https://github.com/JedWatson/classnames)

File: src/containers/MenuMain.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  logoutUser: () => dispatch(logoutUser()),
+  toggleElement: element => dispatch(toggleElement(element))
+})
+
+const MenuMainContainer = connect(
+  mapStateToProps,
+  mapDispatchToProps
+)(MenuMain)
+
+export default MenuMainContainer
```
Comment (sptaszek92) [inline src/containers/MenuMain.js]: again - no need to use connect in const :)

```
export default connect(mapStateToProps, mapDispatchToProps)(MenuMain)
```

File: src/components/Auth.js [hunk 3]
```diff
@@ -1,12 +1,18 @@
-import React from 'react'
+import React, { Component } from 'react'
 import { Redirect } from 'react-router'
+import PropTypes from 'prop-types'
```
Comment (sptaszek92) [inline src/components/Auth.js]: Almost good 😆 
But have you added `prop-types` package via npm ?

Comment (dominikakosiedowska) [reply src/components/Auth.js]: actually I have but on different branch :/

Comment (dominikakosiedowska) [reply src/components/Auth.js]: i fixed it :)

**Pull request merged**